.eleventy.js

```diff
@@ -159,6 +159,79 @@ module.exports = function (config) {
     </svg>`;
   });
 
+  // Set the modal 
+  //
+  // Potential tech debt
+  //
+  // We're hardcoding all this stuff instead of using params to
+  // get this deployed quickly. If we use another modal on the site,
+  // we should consider refactoring this to a paired shortcode with
+  // multiple params. If we use 3+ different modals on the site, we
+  // HAVE to refactor this.
+  config.addLiquidShortcode('modal-start-application', function () {
+    return `<div class="margin-y-3">
+      <a
+        href="#modal-start-application"
+        class="usa-button"
+        aria-controls="modal-start-application"
+        data-open-modal
+      >Start a .gov domain request</a>
+      <div
+        class="usa-modal"
+        id="modal-start-application"
+        aria-labelledby="modal-start-application-heading"
+        aria-describedby="modal-start-application-description"
+      >
+        <div class="usa-modal__content">
+          <div class="usa-modal__main">
+            <h2 class="usa-modal__heading utility-header" id="modal-start-application-heading">
+            Requirements for requesting a .gov domain
+            </h2>
+            <div class="usa-prose">
+              <ul>
+                <li>You must be a government employee, or be working on behalf of a government organization, to request a .gov domain.</li>
+                <li><a href="https://login.gov/help/get-started/create-your-account/">You must have a Login.gov account</a>. Login.gov provides a simple and secure process for signing into many government services with one account.
```

Comment:
**Question:** I couldn't get the full Figma to load bc of access issues, so bear with me on this (see screenshot below of what I've pulled from Mural):

1. It looks like "You must have a Login.gov account" is bolded, but only "Login.gov account" has a link associated with it
2. The text for the white open circle bullet point is different as well with "Read more about why..." than what you have here

Is Mural the correct wording, or would you mind pasting a screenshot of the Figma with the approved wording? Sorry for running you around in circles!

<img width="734" alt="Screenshot 2024-01-12 at 1 57 11 PM" src="https://github.com/cisagov/getgov-home/assets/13954664/b60b9dd1-be6c-4bc4-b0e1-e315cf0af063">

Reply:
The approved wording came from the Google doc linked in the ticket. I'm pretty sure I asked @michelle-rago which text is canonical when I was doing this ticket.